doc-content/enterprise-only/fuse/ba-engines-karaf-con.adoc

```diff
@@ -0,0 +1,16 @@
+[id='ba-engines-karaf-con']
+= {PRODUCT} decision and process engines with {FUSE} on Apache Karaf
+
+Apache Karaf is a polymorphic standalone open-source container. It is based on the OSGi standard from the OSGi Alliance. Karaf provides support for modularisation through OSGi bundles with sophisticated class-loading support. You can deploy multiple versions of a dependency side by side in a Karaf container. You can use hot code swapping to upgrade or replace a module without shutting down the container.
+
+{PRODUCT} integration with {FUSE} on Karaf is provided through Karaf features. You can install individual components of {PRODUCT} for {FUSE} on Karaf using these features.
+
+The following features XML files facilitate {PRODUCT} and {FUSE} on Karaf integration:
```

Comment:
I'm not sure what you mean by "features XML files". I thought it was a typo at first for "following XML files", but then saw it several places coming up. So is that just a common phrase/jargon with Karaf, or even in industry? I've never hear of it and causes me to pause each time I read it.